crates/astria-composer/src/executor/mod.rs

```diff
@@ -293,9 +287,9 @@ impl Executor {
         // sequence actions
         self.serialized_rollup_transactions.close();
 
-        log_exit(&reason);
+        report_exit_reason(reason.as_ref());
```

Comment:
I think you can do this to have a cleaner `report_exit_reason` impl:
```suggestion
        report_exit_reason(reason.as_deref());
```